<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsSchedule</class>
 <widget class="QWidget" name="SettingsSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>366</width>
        <height>549</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupXmltv">
         <property name="title">
          <string>XMLTV settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelLocation">
            <property name="text">
             <string>XMLTV file location:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BrowseWidget" name="location" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;i&gt;All XMLTV settings require restart.&lt;/i&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupRefresh">
         <property name="title">
          <string>Refresh XMLTV</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkRefresh">
            <property name="text">
             <string>Refresh XMLTV file on startup</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioNetwork">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Download file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioGrabber">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Run grabber</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="labelRefresh">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&lt;i&gt;Grabber running is not available yet.&lt;/i&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupUrl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Network</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labelUrl">
            <property name="text">
             <string>XMLTV source url:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editUrl"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupGrabber">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Grabber</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelSelect">
            <property name="text">
             <string>Select desired XMLTV grabber from the list below:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboGrabber"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>93</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>common/BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioNetwork</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
